src/Microsoft.ML.Data/Transforms/NormalizeColumn.cs

```diff
@@ -290,8 +290,8 @@ internal static IDataTransform Create(IHostEnvironment env, MinMaxArguments args
 
             var columns = args.Columns
                 .Select(col => new NormalizingEstimator.MinMaxColumnOptions(
-                    col.Name,
-                    col.Source ?? col.Name,
+                    col.OutputColumnName,
```

Comment:
>OutputColumnName [](start = 24, length = 16)

I don't think that this is necessarily important, but in prior discussions we've discussed that in the case of a column options *specifically*, given that it's clear this is talking about columns we have discussed that it may be wise to name it `InputName` as opposed to `InputColumnName`, since it's obvious we're talking about columns here.